Formula/uriparser.rb

```diff
@@ -1,8 +1,12 @@
 class Uriparser < Formula
   desc "URI parsing library (strictly RFC 3986 compliant)"
   homepage "https://uriparser.github.io/"
-  url "https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.1/uriparser-0.9.1.tar.bz2"
-  sha256 "75248f3de3b7b13c8c9735ff7b86ebe72cbb8ad043291517d7d53488e0893abe"
+
+  stable do
```

Comment:
Please don't encapsulate in a stable block if there's only stable.

Reply:
Forgot to add the Git repo, will do soon.